docs/examples/tf/pets_training/create_pet_tf_record.py

```diff
@@ -0,0 +1,314 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
if this is Apache license, we cannot host this file. We should give a link to the tf repo and ask user to run from there. 
@kiritigowda can you confirm this?

Reply:
The code in this file is copied and modified from this [file](https://github.com/tensorflow/models/blob/master/research/object_detection/dataset_tools/create_pet_tf_record.py) so we no longer need this models package installation which causes GPU issues. I can remove this copyright if needed